package.json

```diff
@@ -9,8 +9,9 @@
 		"email": "sindresorhus@gmail.com",
 		"url": "sindresorhus.com"
 	},
+	"type": "module",
```

Comment:
Don't forget to add `exports`: https://github.com/sindresorhus/meta/discussions/15